Foundry/pf2e-magician/module.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "pf2e-magician",
   "title": "PF2e Magician",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Also, this was by accident. I can revert this if need be.

Reply:
I'll leave it as is and just remember to not increment when packaging.